.goreleaser.yml

```diff
@@ -0,0 +1,59 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+- env:
+  - CGO_ENABLED=0
```

Comment:
I believe this is here because goreleaser doesn't support CGO: https://goreleaser.com/cgo/

We could remove this, but people may be surprised when they attempt to build with it.

Reply:
Oh good call, fine with me to leave it.